examples/sapiaPicture/default-pjs.js

```diff
@@ -39,18 +39,19 @@ var run;
       var tempContext = canvas.getContext("2d");
 
       tempContext.drawImage(source, 0, 0, canvas.width, canvas.height);
-
       var canvasData = tempContext.getImageData(0, 0, canvas.width, canvas.height);
+      var temp = new SharedUint8ClampedArray(canvasData.data.length);
```

Comment:
there should be a separate samples for shared array buffers and normal array buffers. just copy paste and change the array types.